checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/AccessCheckoutCard.kt

```diff
@@ -135,21 +152,64 @@ class AccessCheckoutCard @JvmOverloads constructor(
 
 }
 
+/**
+ * [CardFactory] is responsible for constructing the dependencies needed for a [Card]
+ */
 interface CardFactory {
 
+    /**
+     * Creates a [CardValidator] instance
+     * @param cardConfiguration the card configuration to use
+     * @return [CardValidator] for validating the inputs of a card
+     */
     fun getCardValidator(cardConfiguration: CardConfiguration): CardValidator
+
+    /**
+     * Creates a [CardConfiguration] instance
+     * @return [CardConfiguration] for holding configurations for validating a card
+     */
     fun getCardConfiguration(): CardConfiguration
+
+    /**
+     * Creates a [PANLengthFilter] instance
+     * @param (Optional)[cardValidator] the card validator to use
+     * @param cardConfiguration the card configuration to use
+     * @return (Optional)[PANLengthFilter] for restricting the inputs of a pan field
+     */
     fun getPANLengthFilter(cardValidator: CardValidator?, cardConfiguration: CardConfiguration): PANLengthFilter?
+
+    /**
+     * Creates a [CVVLengthFilter] instance
+     * @param (Optional)[cardValidator] the card validator to use
+     * @param cardConfiguration the card configuration to use
+     * @param panView the pan field so that the cvv length can be validated against the current pan
+     * @return (Optional)[CVVLengthFilter] for restricting the inputs of a cvv field
```

Comment:
* @return [CVVLengthFilter] (Optional) for restricting the inputs of a cvv field